test011.c:35: エラー: `t' が再宣言されています.
test011.c:34: エラー: ここで宣言されていました.
